mitosheet/css/toolbar.css

```diff
@@ -181,4 +181,21 @@
   padding: 0px 5px;
   color: var(--mito-background);
   height: 18px;
+}
+
+
+/* Tab Bar Style */
+.mito-toolbar-tabbar-button {
+  border: none;
+  background: none;
+  color: var(--mito-text-medium);
+}
+
+.mito-toolbar-tabbar-button-selected span {
+  border-bottom: 3px solid var(--mito-highlight);
```

Comment:
In the mockups, we round the corners of these. I think it's a nice bit of polish.